core/src/main/java/org/apache/metamodel/CompositeDataContext.java

```diff
@@ -190,9 +183,7 @@ public Schema getSchemaByNameInternal(String name) throws MetaModelException {
                 }
             }
         }
-        String[] result = set.toArray(new String[set.size()]);
-        Arrays.sort(result);
-        return result;
+        return new ArrayList<>(set);
```

Comment:
Doesn't order still need to be preserved here, at least to keep the current behavior the same?